build.yaml

```diff
@@ -900,6 +900,22 @@ libs:
   - grpc++_codegen
   secure: check
   vs_project_guid: '{C187A093-A0FE-489D-A40A-6E33DE0F9FEB}'
+- name: grpc++_reflection
+  build: all
+  language: c++
+  public_headers:
+  - extensions/include/grpc++/impl/proto_server_reflection_plugin.h
```

Comment:
The public headers will be installed to users machine when make install is called. For example now our headers could be at /usr/local/include/grpc++/. By adding "extentions/" in front, the new headers would be /usr/local/extensions/include/grpc++. I suspect we should put things as include/grpc++/impl/extentions/.